lib/internal/bootstrap_node.js

```diff
@@ -327,6 +328,24 @@
     setupInspector(originalConsole, wrappedConsole, Module);
   }
 
+  function setupGlobalURL() {
+    const { URL, URLSearchParams } = NativeModule.require('internal/url');
```

Comment:
Does this impact startup performance, and if so, could it be required on use possibly?

Reply:
No, as this module is already being imported on startup:

https://github.com/nodejs/node/blob/a3555d0583452bec05e339bf2872cbb4e38236c9/lib/internal/bootstrap_node.js#L80-L82